rescript/ncbi.py

```diff
@@ -6,16 +6,31 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
+import sys
 import time
-import warnings
+import logging
+import json
 
 import requests
+from requests.exceptions import (
```

Comment:
We should list `requests` as a conda dep here:

https://github.com/bokulich-lab/RESCRIPt/blob/master/ci/recipe/meta.yaml

Reply:
Hey @misialq, we will need to address this before we can create our next release. Want me to open a PR, or do you want to deal with this? Either way is good with me, just let me know!

Reply:
oh sorry @thermokarst that was my fault... I asked @misialq to merge but somehow did not see your comment, did not get a notification!

if you want to cut a PR today that would be great, otherwise would you mind opening an issue and one of us can grab it next week?

Reply:
Ouch, sorry @thermokarst ! I missed your last comment. As an exercise, I can try to do that - I'll reach out should there be an issue - is that ok?

Reply:
Sounds like a plan, let me know if you need anything!